src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -4,6 +4,8 @@ Release History
 ===============
 0.2.11
 ++++++
+* webapp: az webapp up is updated to create a P1V1 SKU app if os is Linux
```

Comment:
This could be updated to something like:
"az webapp up is updated to create a P1V1 SKU app if os is Linux and **--sku is unspecified**" ?
or something like 
"az webapp up --sku - defaults to P1V1 if the os is linux."